connectors/__init__.py

```diff
@@ -5,5 +5,17 @@
 #
 import os
 
+import yaml
+
 with open(os.path.join(os.path.dirname(__file__), "VERSION")) as f:
     __version__ = f.read().strip()
+
+yaml_path = os.path.join(os.path.dirname(__file__), "build.yaml")
```

Comment:
For our sanity later, can you add a comment here with an explanation of where this file comes from? Maybe even a link to this PR?